docs/how-to/authentication-plugin-examples.md

```diff
@@ -5,41 +5,41 @@ slug: /plugins/guides/authentication
 pagination_next: getting-started/authentication
 ---
 
-# How To Update Your Authentication and Authorization Setup
+# Handle Authentication and Authorization in your Service
 
 Authentication and authorization are important parts of your Amplication service. Amplication provides flexible options for implementing these security measures.
 
 This guide will walk you through the process of updating your authentication and authorization setup in your Amplication-generated service.
 
-## Creating a Service Without Authentication
+## Adding Authentication to Your Service
 
 If you initially created your service without authentication, you can still add it later. Here are the key steps:
 
 1. Go to your service's Plugins page.
-2. Choose and add an Auth Provider plugin (e.g., JWT, Auth0, etc.) that suits your needs.
-3. Configure the Authentication Entity in your service settings.
+2. Navigate to the "Authentication" category in the left sidebar.
+3. Choose and add an Auth Provider plugin (e.g., JWT, Auth0, etc.) that suits your needs.
+4. Install the "NestJS Auth Module" plugin if it's not already installed. It's a pre-requisite if you're adding Authentication to a Node.js service.
```

Comment:
```suggestion
4. In case of adding Authentication to a Node.js service, make sure to Install the "NestJS Auth Module" plugin.
```

Reply:
I think this line- i.e. step # 4- should be before # 3 above.
In addition- do we need to add a note regarding the "User" entity? [The "User" entity is relevant for both .NET and Node.js? No so sure about it- can you check please?]